### PR TITLE
feat: [SRM-14205]: Request based SLO details page

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

Screenshot 2023-03-29 at 8 28 42 AM

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/slos/SLOCard/SLOCardContent.tsx [hunk 1]
```diff
@@ -189,21 +194,23 @@ const SLOCardContent: React.FC<SLOCardContentProps> = props => {
...
+                </Heading>
+                <ErrorBudgetGauge customChartOptions={getErrorBudgetGaugeOptions(serviceLevelObjective)} />
+                <Text
+                  font={{ variation: FontVariation.SMALL }}
+                  className={css.errorBudgetRemaining}
+                  width={175}
+                  data-testid="errorBudgetRemaining"
+                >
+                  {serviceLevelObjective.errorBudgetRemaining}
+                  <span style={{ display: 'block' }}>{getString('cv.minutesRemaining')}</span>
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/SLOCard/SLOCardContent.tsx:210]: if we are giving `span` and style as block, we can instead use `div` right?

File: src/modules/10-common/components/Stepper/components/Step/components/StepTitle/StepTitle.tsx [hunk 2]
```diff
@@ -45,7 +45,13 @@ export const StepTitle = ({
             className={cx(css.iconNoBorder, !isErrorOrSuccess && css.editIcon)}
           />
         ) : (
-          <Icon name={icon} size={20} margin="small" color={iconColor} className={css.iconNoBorder} />
+          <Icon
+            name={icon}
+            size={20}
+            margin="small"
+            color={iconColor}
+            className={cx(css.iconNoBorder, !isErrorOrSuccess && css.defaultState)}
```
Comment (praneshg239) [inline src/modules/10-common/components/Stepper/components/Step/components/StepTitle/StepTitle.tsx:53]: we can write this as 
```ts
className={cx(css.iconNoBorder, {
[css.defaultState]: !isErrorOrSuccess
})}
```

File: src/modules/85-cv/pages/slos/SLOCard/SLOCardContent.tsx (hunk 1 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/slos/SLOCard/SLOCardContent.tsx:210]: Older code , just wrapped inside hideErrorBudgetGauge

File: src/modules/10-common/components/Stepper/components/Step/components/StepTitle/StepTitle.tsx (hunk 2 above)
Comment (praneshg239) [reply src/modules/10-common/components/Stepper/components/Step/components/StepTitle/StepTitle.tsx:53]: Please take it up in next PR @deepesh-ui

**Review by praneshg239 — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

File: src/modules/10-common/components/Stepper/components/Step/components/StepTitle/StepTitle.tsx (hunk 2 above)
Comment (deepesh-ui) [reply src/modules/10-common/components/Stepper/components/Step/components/StepTitle/StepTitle.tsx:53]: sure

**Pull request merged**